docs/webhooks/03-Mutual-TLS.md

```diff
@@ -21,10 +21,10 @@ These steps assume you already have server authentication setup.
 
 In general, there are five steps needed to turn on client authentication for your server:
 
-1. Download the PEM version of the [DigiCert Global Root CA](https://cacerts.digicert.com/DigiCertGlobalRootCA.crt.pem) certificate.
+1. Download the PEM version of the DigiCert Global Root G2 certificate from our [Public Certificates Page](../../docs/webhooks/08-Certificates.md).
```

Comment:
nit: unlikely this will change but we could be even more non-specific by saying "Download the PEM version of the DigitCert root certificate..."